libs/pages/application/src/lib/ui/page-settings-healthchecks/page-settings-healthchecks.tsx

```diff
@@ -22,32 +25,36 @@ export function PageSettingsHealthchecks({
   loading,
   isJob,
   jobPort,
-  linkPortSetting,
+  linkResourcesSetting,
   defaultTypeReadiness,
   defaultTypeLiveness,
+  maxRunningInstances,
 }: PageSettingsHealthchecksProps) {
   const { formState } = useFormContext()
+  const { projectId = '', environmentId = '' } = useParams()
+  const { data: environments = [] } = useFetchEnvironments(projectId)
+  const environment = getEnvironmentById(environmentId ?? '', environments)
 
   return (
     <div className="flex flex-col justify-between w-full text-ssm">
       <div className="p-8 max-w-content-with-navigation-left">
-        {ports && ports?.length === 0 && (
+        {environment?.mode === EnvironmentModeEnum.PRODUCTION && maxRunningInstances === 1 && (
           <BannerBox
             className="mb-2"
-            dataTestId="banner-box-port-configuration-required"
-            title="Port configuration required"
-            type={BannerBoxEnum.WARNING}
             message={
-              <>
-                Please configure a port before using any health check types, except for the EXEC type.{' '}
+              <span>
+                Your service is configured to run with a minimum of one instance, setting the health checks will not
+                ensure the service high availability during a cluster upgrade. Have a look at your{' '}
                 <Link
-                  className="link !block text-accent2-500 mt-1"
+                  className="link text-accent2-500 mt-1"
```

Comment:
I think you can remove the `mt-1` to remove the UI offset

Reply:
Fixed
https://github.com/Qovery/console/pull/680/commits/6c3f7f6f25ff86ed40fc4362f42a5ea35041749c